packages/launchpad/cypress/e2e/choose-a-browser.cy.ts

```diff
@@ -1,6 +1,12 @@
 import type { FoundBrowser } from '@packages/types'
 
 describe('Choose a browser page', () => {
+  before(() => {
+    cy.withCtx((ctx, _) => {
+      ctx.actions.project.launchCount = 0
```

Comment:
I have to set the launch count here before the tests start. I don't feel this is the best way to test it, but I'm out of ideas to get this going.